src/views/Generator/GeneratorTabs/GeneratorTabs.tsx

```diff
@@ -77,6 +80,20 @@ export function GeneratorTabs({
                 )}
               </Flex>
               <Flex pr="2" pl="4" gap="4">
+                <Text
+                  as="label"
+                  size="1"
+                  color={!rulesEnabled ? 'gray' : undefined}
+                >
+                  <Flex gap="2">
+                    <Switch
+                      size="1"
+                      checked={rulesEnabled}
+                      onCheckedChange={setEnableRules}
+                    />
+                    Apply rules
```

Comment:
To me, this part of the new UI is rather confusing - why would I not want to apply rules to my recording?
I also think that when it comes to script generation rules should be applied regardless of this setting.

I don't have a good copy suggestion, but maybe something focused on the recording rather than rules? The reason I think it might be better is because the goal of having this switch (as far as I understand) is for the user to be able to check the original recording without any test rules applied.

Reply:
I've used "View original requests" and will revisit after we collect feedback